### PR TITLE
Add Brew Cask Upgrade Command

### PR DESCRIPTION
**Pull request by LanikSJ**
## Description

Add Brew Cask Upgrade Command

## Type of change

- [x] New script command

## Screenshot

Screen Shot 2022-03-01 at 2 48 33 PM

## Dependencies / Requirements

Installation [cask upgrade](https://github.com/buo/homebrew-cask-upgrade) is required.

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/developer-utils/brew/brew-cask-upgrade.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
```
Comment (dehesa) [inline commands/developer-utils/brew/brew-cask-upgrade.sh:2]: If a dependency is required, as it's in your case, you need to define it at the top of the file, right under the `!#/bin/bash`. Check out the [CONTRIBUTING.md](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md) 😉 

```bash
# Dependency: This script requires `brew-cask-upgrade` cli installed: https://github.com/buo/homebrew-cask-upgrade
# Install via homebrew: `brew tap buo/cask-upgrade `
```

File: commands/developer-utils/brew/brew-cask-upgrade.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# @raycast.schemaVersion 1
+# @raycast.title Cask Upgrade
+# @raycast.mode fullOutput
+# @raycast.packageName Brew
+#
+# Optional parameters:
+# @raycast.icon 🍺
+#
+# Documentation:
+# @raycast.description Run Brew Cask Upgrade
```
Comment (dehesa) [inline commands/developer-utils/brew/brew-cask-upgrade.sh:13]: The description should be a bit more "descriptive". Such as: `Updates brew and upgrades every outdate app.`

**Review by dehesa — CHANGES_REQUESTED** [review]
Hey @LanikSJ,

Thank you for your contribution. I didn't know about [`brew-cask-upgrade`](https://github.com/buo/homebrew-cask-upgrade), but it looks pretty neat.

I left you some small comments/suggestions.

File: commands/developer-utils/brew/brew-cask-upgrade.sh (hunk 1 above)
Comment (LanikSJ) [reply commands/developer-utils/brew/brew-cask-upgrade.sh:2]: I have no idea how but I totally missed that text I will address it.

File: commands/developer-utils/brew/brew-cask-upgrade.sh (hunk 2 above)
Comment (LanikSJ) [reply commands/developer-utils/brew/brew-cask-upgrade.sh:13]: Thank you will make this change also.

**Comment by dehesa** [comment]
Looks great! Thanks again and congrats on your 1st contribution! 🎉

**Pull request merged**